sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 ### Bugs Fixed
 - Adds missing API version `2022-12-01` to the list of supported API versions.
+- Fixed list_routes and list_trunks functions on SIP routing client to return (Async)ItemPaged object.
```

Comment:
I'm not sure this should be considered a bug fix. I'm leaning more towards "Other Changes" or "Breaking Changes", although I don't know if it's considered a breaking change since the previous revision was never in GA. @annatisch, do you have any comments on this?

Reply:
Good question, I've updated it as Other Changes, seemed most appropriate.

Reply:
I don't feel too strongly on it - as you say it was never GA :)
"other changes" seems good to me.